src/main/java/teammates/ui/controller/InstructorFeedbackQuestionEditAction.java

```diff
@@ -80,46 +80,30 @@ private void editQuestion(FeedbackQuestionAttributes updatedQuestion) throws Inv
             statusToUser.add(new StatusMessage(err, StatusMessageColor.DANGER));
             isError = true;
         }
-        
-        if (updatedQuestion.questionNumber != 0) { // Question number was updated
-            List<String> questionDetailsErrors = updatedQuestion.getQuestionDetails().validateQuestionDetails();
-            List<StatusMessage> questionDetailsErrorsMessages = new ArrayList<StatusMessage>();
-            
-            for (String error : questionDetailsErrors) {
-                questionDetailsErrorsMessages.add(new StatusMessage(error, StatusMessageColor.DANGER));
-            }
-            
-            // if error is not empty not tested as extractFeedbackQuestionData method uses Assumptions to cover it
-            if (!questionDetailsErrors.isEmpty()) {
-                statusToUser.addAll(questionDetailsErrorsMessages);
-                isError = true;
-            } else {
-                logic.updateFeedbackQuestionNumber(updatedQuestion);
-                statusToUser.add(new StatusMessage(Const.StatusMessages.FEEDBACK_QUESTION_EDITED, StatusMessageColor.SUCCESS));
-            }
+
+        List<String> questionDetailsErrors = updatedQuestion.getQuestionDetails().validateQuestionDetails();
+        List<StatusMessage> questionDetailsErrorsMessages = new ArrayList<StatusMessage>();
+
+        for (String error : questionDetailsErrors) {
+            questionDetailsErrorsMessages.add(new StatusMessage(error, StatusMessageColor.DANGER));
+        }
+
+        // if error is not empty not tested as extractFeedbackQuestionData
+        // method uses Assumptions to cover it
+        if (!questionDetailsErrors.isEmpty()) {
+            statusToUser.addAll(questionDetailsErrorsMessages);
+            isError = true;
         } else {
-            List<String> questionDetailsErrors = updatedQuestion.getQuestionDetails().validateQuestionDetails();
-            List<StatusMessage> questionDetailsErrorsMessages = new ArrayList<StatusMessage>();
+            logic.updateFeedbackQuestionNumber(updatedQuestion);
```

Comment:
Hmm, I meant the entire codebase. If <code>updateFeedbackQuestion</code> is not used anywhere in the code, we can remove it and rename <code>updateFeedbackQuestionNumber</code> to updateFeedbackQuestion. Will reduce code duplication, since currently updateFeedbackQuestionNumber and updateFeedbackQuestion are very similar anyway. Could be done in a separate issue if it's too big a change though.